src/Whoops/Run.php

```diff
@@ -395,6 +410,13 @@ private function writeToOutputNow($output)
         return $this;
     }
 
+    private static function isInternalErrorException($exception)
+    {
+        return class_exists('Error') &&
+            !is_subclass_of('Error', 'Exception') &&
```

Comment:
I guess you are trying to check whether the defined Error class was actually provided by php-src instead of a lib/framework, right?